docs/modules/ROOT/pages/extensions/threadpoolfactory-vertx.adoc

```diff
@@ -0,0 +1,28 @@
+// Do not edit directly!
+// This file was generated by camel-quarkus-maven-plugin:update-extension-doc-page
+
+[[threadpoolfactory-vertx]]
+= ThreadPoolFactory Vert.x
+
+[.badges]
+[.badge-key]##Since Camel Quarkus##[.badge-version]##1.0.0-CR4## [.badge-key]##JVM##[.badge-supported]##supported## [.badge-key]##Native##[.badge-supported]##supported##
+
+ThreadPoolFactory for camel-core using Vert.x
+
+== What's inside
+
+* https://camel.apache.org/components/latest/others/threadpoolfactory-vertx.html[ThreadPoolFactory Vert.x]
```

Comment:
threadpoolfactory-vertx.html: is it https://camel.apache.org/components/latest/others/reactive-threadpoolfactory-vertx.html ?

Reply:
Filed a followup https://github.com/apache/camel-quarkus/issues/1506